packages/driver/src/cypress/chai_jquery.coffee

```diff
@@ -172,15 +183,15 @@ $chaiJquery = (chai, chaiUtils, callbacks = {}) ->
       else
         _super.apply(@, arguments)
 
-  _.each selectors, (selector) ->
+  _.each _.keys(selectors), (selector) ->
```

Comment:
very nice

Reply:
instead of using `_.each` and `_.keys` ... just use `_.each` which yields you `(value, key)` directly.

Reply:
smart